docs/definitionOfReady.md

```diff
@@ -0,0 +1,13 @@
+# Definition of Ready
```

Comment:
The same applies here, should the Definition of Ready not be the list of items we agreed as a team from the Github issues, if there are any additional criteria we will need to discuss as a team.

Reply:
Please write what are the items you have an objection about.